lib/src/admob.dart

```diff
@@ -5,4 +5,9 @@ class Admob {
     MethodChannel _channel = const MethodChannel('admob_flutter');
     _channel.invokeMethod('initialize', appId);
   }
+
+  Admob.addTestDevice(String deviceId) {
+    MethodChannel _channel = const MethodChannel('admob_flutter');
+    _channel.invokeMethod('addTestDevice', deviceId);
```

Comment:
Maybe I'm missing it but I don't see where this is implemented on the native side?